.github/workflows/build.yml

```diff
@@ -136,7 +133,7 @@ jobs:
 
       - name: Build the PHAR
         run: |
-          composer config platform.php 7.2.34
+          composer config platform.php 8.2.25
```

Comment:
mostly used version of the lowest non-EOL version to date

Reply:
This was only there as a workaround to allow box (which I think requires ^8.2) to run but build with deps that were valid for 7.2.

It would be better to remove this and change the setup-php step to run on 8.2.  As we are now keeping pace with PHP version support, we should always be able to find a version of box that will run on our lowest supported version, so we won't need this workaround in future.